.github/workflows/releaseinfo.yml

```diff
@@ -0,0 +1,26 @@
+name: write release information to file
+
+on: workflow_dispatch
+
+jobs:
+  release_info:
+    runs-on: ubuntu-latest
+    steps:
+      - id: info
+        uses: HarikrishnanBalagopal/nextrelease@v1
```

Comment:
This repo should be in Konveyor org. Can we name it move2kube-nextrelease?

Reply:
Doesn't that require a lot of approvals?

Reply:
No. Only if we need to use generic names or temp repo we need to ensure there is a valid reason.

Reply:
So can you create the repo? I will upload the code there.

Reply:
Sure, Name? move2kube-nextreleaseinfo-action?